src/diet.ts

```diff
@@ -318,7 +318,7 @@ export function runDiet(): void {
       };
     });
     const best = chocoVals.sort((a, b) => b.value - a.value)[0];
-    acquire(1, best.choco, best.value, false);
+    acquire(1, best.choco, best.value + get("valueOfAdventure"), false);
```

Comment:
Shouldn't it add back `mallPrice(best.choco)`, since that is what's subtracted when calculating the best choco?

Reply:
I think in retrospect what this means is that we shouldn't use acquire at all

Reply:
yeah i'm very dumb